<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnaProfils</class>
 <widget class="QWidget" name="AnaProfils">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>257</width>
    <height>102</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="gb_Name">
     <property name="title">
      <string>Nom</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="l_Duree">
        <property name="text">
         <string>Dur√©e totale d'acquisition</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="l_Nombre">
        <property name="text">
         <string>Nombre de profils</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="l_Longueur">
        <property name="text">
         <string>Longueur totale des profils</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
